pkgs/development/python-modules/glob2/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "glob2";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "85c3dbd07c8aa26d63d7aacee34fa86e9a91a3873bc30bf62ec46e531f92ab8c";
```

Comment:
Should be base32

Reply:
Note we have not documented this yet. I am not doing it yet either because it's always a bit of a hassle even though I agree we should go for base32.
https://github.com/ryantm/nixpkgs-update/issues/106

Reply:
Fixed.